app/models/vote.rb

```diff
@@ -125,6 +125,11 @@ def self.maximum_vote
     MAXIMUM_VOTE
   end
 
+  # minimum owner's vote needed to display owner_id
+  def self.owner_id_min_confidence
+    min_pos_vote
```

Comment:
Also seems odd to me.  I prefer the functional approach.  
But as fixing this will take changes to multiple files and it's not a PR blocker, I will leave as-is and create a Pivotal story.